plugins/analytics-provider-segment/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "@janus-idp/plugin-analytics-provider-segment",
```

Please change to maintain naming convention

```suggestion
  "name": "@janus-idp/backstage-plugin-analytics-provider-segment",
```

or 

```suggestion
  "name": "@janus-idp/backstage-plugin-segment",
```

---

I think the original author just followed the convention that was used with the Google Analytics plugin upstream. I am OK with the change but we need to update the README etc too